### PR TITLE
style: explain other possible values

### PR DESCRIPTION
**Pull request by noraj**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

let me know if its correct before I do the change for other languages

### CODE REVIEW

**Comment by noraj** [comment]
Someone to review this before I commit for all other languages?

File: source/docs/helpers.md [hunk 1]
```diff
@@ -563,7 +563,7 @@ Option | Description | Default
 `orderby` | Order of categories | name
 `order` | Sort of order. `1`, `asc` for ascending; `-1`, `desc` for descending | 1
 `show_count` | Display the number of posts for each category | true
-`style` | Style to display the category list. `list` displays categories in an unordered list.  | list
+`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value  to disable it. | list
```
Comment (stevenjoezhang) [inline source/docs/helpers.md:566]:
```suggestion
`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value to disable it. | list
```

File: source/docs/helpers.md [hunk 2]
```diff
@@ -601,8 +601,7 @@ Option | Description | Default
 `orderby` | Order of categories | name
 `order` | Sort of order. `1`, `asc` for ascending; `-1`, `desc` for descending | 1
 `show_count` | Display the number of posts for each tag | true
-`style` | Style to display the tag list. `list` displays tags in an unordered list.  | list
-`separator` | Separator between categories. (Only works if `style` is not `list`) | ,
+`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value  to disable it. | list`separator` | Separator between categories. (Only works if `style` is not `list`) | ,
```
Comment (stevenjoezhang) [inline source/docs/helpers.md:604]:
```suggestion
`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value to disable it. | list`separator` | Separator between categories. (Only works if `style` is not `list`) | ,
```

File: source/docs/helpers.md [hunk 3]
```diff
@@ -622,7 +621,7 @@ Option | Description | Default
 `order` | Sort of order. `1`, `asc` for ascending; `-1`, `desc` for descending | 1
 `show_count` | Display the number of posts for each archive | true
 `format` | Date format | MMMM YYYY
-`style` | Style to display the archive list. `list` displays archives in an unordered list.  | list
+`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value  to disable it. | list
```
Comment (stevenjoezhang) [inline source/docs/helpers.md:624]:
```suggestion
`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value to disable it. | list
```

File: source/docs/helpers.md [hunk 4]
```diff
@@ -639,7 +638,7 @@ Option | Description | Default
 --- | --- | ---
 `orderby` | Order of posts | date
 `order` | Sort of order. `1`, `asc` for ascending; `-1`, `desc` for descending | 1
-`style` | Style to display the post list. `list` displays posts in an unordered list.  | list
+`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value  to disable it. | list
```
Comment (stevenjoezhang) [inline source/docs/helpers.md:641]:
```suggestion
`style` | Style to display the category list. `list` displays categories in an unordered list. Use `false` or any other value to disable it. | list
```

**Pull request merged**